CHANGELOG.md

```diff
@@ -78,3 +78,19 @@ the following form: YYYY.0M.0D.
 - Designed a Google Firebase Firestore component
 - Designed a Shopping Cart component
 - Designed a Bank account component
+
+## [2024.10.03]
```

Comment:
Put the latest changes at the top of this file. It probably seems weird, but it's so the latest changes are the first thing you see.